src/VKID/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "socialiteproviders/vkid",
+    "description": "VK ID OAuth2 Provider for Laravel Socialite",
+    "keywords": [
+        "laravel",
+        "oauth",
+        "provider",
+        "socialite",
+        "vkontakte",
+        "vkid"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Vladimir Kharinenkov",
+            "email": "vhar@mail.ru"
+        }
+    ],
+    "require": {
+        "php": "^7.4 || ^8.0",
```

Comment:
non php 7, lets limit to 8.4+

Reply:
https://endoflife.date/php
We can move to 8.2, but all other are basically out of sec support, and long out of active maintenance